tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedReadHandleImplV2.java

```diff
@@ -196,6 +209,10 @@ public LedgerMetadata getLedgerMetadata() {
                             val skipped = groupedReader.inputStream.skip(length);
                         }
                     }
+                    if (entries.isEmpty()) {
+                        ByteBuf buf = PooledByteBufAllocator.DEFAULT.buffer(0, 0);
+                        entries.add(LedgerEntryImpl.create(ledgerId, 0, 0, buf));
```

Comment:
we can't do this, because `LedgerEntryImpl.create(ledgerId, 0, 0, buf)` will be sent to consumer but it don't have any data